airbrake/notifier.py

```diff
@@ -186,15 +186,13 @@ def log(self, exc_info=None, message=None, filename=None,
                    'notifier': self.notifier,
                    'environment': environment,
                    'session': session}
-
-        return self.notify(payload)
+        return self.notify(json.dumps(payload, cls=utils.FailProofJSONEncoder))
```

Comment:
@samstav lgtm and works nicely in my local environment.

@vmihailenco if you have a moment PTAL.